mifospay/src/main/res/layout/fragment_transfer.xml

```diff
@@ -91,6 +91,7 @@
                           android:layout_width="match_parent"
                           android:layout_height="wrap_content"
                           android:hint="@string/virtual_payment_address"
+                          android:inputType="textEmailAddress"
```

Comment:
Based on my understanding, a VPA looks more like a username than an email address. So, this inputType wouldn't really help our case. We should instead set the `inputType` to `text`.